client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -74,13 +73,12 @@ InlineHelpSearchCard.propTypes = {
 	location: PropTypes.string,
 };
 
-const mapStateToProps = ( state, ownProps ) => ( {
-	isSearching: isRequestingInlineHelpSearchResultsForQuery( state, ownProps.query ),
-	selectedLink: getInlineHelpCurrentlySelectedLink( state ),
```

Comment:
This `selectedLink` prop got removed because it was unused even before this patch?

Reply:
Yes, iirc there were a few occurrences of selectors and/or actions which weren't used at all.